.github/workflows/main.yml

```
@@ -88,25 +88,13 @@ jobs:
     - run: pip install -vvv ${{ steps.create_dist.outputs.whl }}
     - run: rsconnect version
     - run: rsconnect --help
-    - if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
-      id: create_release
-      uses: actions/create-release@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        tag_name: ${{ github.ref }}
-        release_name: Release ${{ github.ref }}
-        draft: false
-        prerelease: false
-    - if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
-      uses: actions/upload-release-asset@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: release
+      uses: softprops/action-gh-release@v2
+      if: startsWith(github.ref, 'refs/tags/')
```

I am not sure if this matters, but I noticed it in the previous iteration:

```suggestion
      if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```